src/main/java/app/model/Label.java

```diff
@@ -0,0 +1,36 @@
+package app.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.Table;
+import java.util.List;
+import lombok.Data;
+import org.hibernate.annotations.SoftDelete;
+
+@Entity
+@Table(name = "labels")
+@SoftDelete(columnName = "is_deleted")
+@Data
```

Comment:
Maybe better to use only setter and getter annotations? It could cause problems with task IDs.